themes/root/theme.config.php

```diff
@@ -99,8 +99,13 @@
             'VuFind\View\Helper\Root\Url' => 'VuFind\View\Helper\Root\UrlFactory',
             'VuFind\View\Helper\Root\UserList' => 'VuFind\View\Helper\Root\UserListFactory',
             'VuFind\View\Helper\Root\UserTags' => 'VuFind\View\Helper\Root\UserTagsFactory',
+
+            'Laminas\View\Helper\EscapeHtmlAttr' => 'VuFind\View\Helper\Root\EscapeHtmlAttrFactory',
```

Comment:
It would have been nice to use HelperInitializer also with EscapeHtmlAttr, but unfortunately the setEscaper method for escaping helpers is deprecated.